.travis.yml

```diff
@@ -32,13 +37,9 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git $HOME/maintainer-quality-tools
```

Comment:
Could you please add ```--depth=1```?
See https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L26 as reference.